bench/bench-local.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh -e
+
+WRKDIR=`realpath $1`/bench
+
+start_kvlds () {
+	mkdir $WRKDIR
+	../lbs/lbs -d $WRKDIR -s $WRKDIR/sock_lbs -b 2048 -p $WRKDIR/lbs.pid
+	../kvlds/kvlds -s $WRKDIR/sock_kvlds -l $WRKDIR/sock_lbs -p $WRKDIR/kvlds.pid
+}
+stop_kvlds () {
+	kill `cat $WRKDIR/kvlds.pid`
+	kill `cat $WRKDIR/lbs.pid`
+	rm -rf $WRKDIR
+}
+
+if [ $# -ne 1 ]; then
+	echo "usage: bench-local.sh WRKDIR"
+	exit 1
+fi
+
+# Testing bulk insert performance
+start_kvlds
+mkpairs/mkpairs `expr 1024 \* 1024 \* 130` > $WRKDIR/pairs
```

Comment:
I napkin this at 70 GB.  Is that right?  1024*1024*130 * 256 * 2 ?  (keyvals are 256 each, and there's 2 of them)
I cancelled my first attempt at running this when the file was over 6 GB.

Do you want to be recreating this file every time you run the benchmark?

Reply:
Correction: these only use 40-byte keyvals, so I'm expecting that this file would be 11 GB.

Reply:
Yes, I want to create that file every time the benchmarks run.  I certainly don't want to keep it sitting around eating up disk space!